javascript/features/houses/house_manager.js

```diff
@@ -249,8 +251,27 @@ class HouseManager {
                 }
 
                 await this.database_.updateHouseAccess(location, value);
+                
+                let readableAccess = 'unknown';
+                switch(value) {
+                    case HouseSettings.ACCESS_EVERYBODY:
+                        readableAccess = 'everybody';
+                        break;
+                    case HouseSettings.ACCESS_FRIENDS_AND_GANG:
+                    readableAccess = 'friends and gangs';
+                        break;
+                    case HouseSettings.ACCESS_FRIENDS:
+                    readableAccess = 'friends';
+                        break;
+                    case HouseSettings.ACCESS_PERSONAL:
+                    readableAccess = 'personal';
```

Comment:
nit: indent on lines 261, 264 and 267

Reply:
Good point; I'll look into my VS code settings. I'm used to that being done for me. #lazy